### PR TITLE
EQL: import QL into EQL

### PR DESCRIPTION
**Pull request by costin**
Link QL into the new build file
Remove duplicate classes and use the new ql package
Update Exception hierarchy on top of QlException

### CODE REVIEW

**Review by aleksmaus — APPROVED** [review]
Thank you!

**Review by rw-access — APPROVED** [review]

File: x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/SqlIllegalArgumentException.java [hunk 1]
```diff
@@ -1,9 +1,9 @@
-package org.elasticsearch.xpack.eql;
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+package org.elasticsearch.xpack.eql;
 
 public class SqlIllegalArgumentException extends EqlServerException {
```
Comment (imotov) [inline x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/SqlIllegalArgumentException.java]: Should this be `EqlIllegalArgumentException`?

**Review by imotov — APPROVED** [review]
LGTM. I think one of the files was not renamed or was moved accidentally (since it is not used by anything). Adding some javadocs to indicate the purpose of different exception would also help. For example how `EqlClientException` vs`EqlServerException` would be very helpful.

File: x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/SqlIllegalArgumentException.java (hunk 1 above)
Comment (costin) [reply x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/SqlIllegalArgumentException.java]: Not sure how this file got it - there's already `EqlIllegalArgumentException `.
Thanks for noticing it - I'll remove it.

**Pull request merged**